client/src/CSS/App.css

```diff
@@ -0,0 +1,227 @@
+/* I got the Nav Bar to persist across pages, after remembering how to import it properly. */
+#navigation {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    margin: 20px 20px 20px 20px;
+}
+
+button {
+    background-color: rgb(242, 151, 118);
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    font-size: 20px;
+}
+
+button:link {
+    text-decoration:  none;
+}
+
+button:hover {
+    color: gold;
+}
+
+.home-wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 100vw;
+    justify-content: center;
+    align-items: center;
+}
+/* It's entirely possible some of this CSS overalaps, and could be better-targeted. */
+#socialMedia {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+#socialMedia a {
+    text-decoration: none;
+    color: black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 20px;
+}
+
+#copyright {
+    display: flex;
+    justify-content: space-evenly;
+}
+
+.image-wrapper {
+    margin: 50px 10px 50px 10px;
+    display: flex;
+    
+}
+
+.image-wrapper h3 {
+    margin-top: 50px;
+    height: 100px;
+}
+/* Working with Michael to refine my CSS skills is ongoing. */
+.projects-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-top: 100px;
+    width: 100vw;
+    justify-content: center;
+    align-items: center;
+}
+
+#hobbies-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+}
+
+#hobbies-wrapper h3{
+    text-align: center;
+    font-size: 28px;
+}
+
+#profilePic {
+    float: left;
+    margin: 50px 15px 30px 0px;
+}
+
+.its-me {
+    display: flex;
+    flex-direction: column;
+}
+
+.its-me a{
+    text-decoration: none;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-weight: bold;
+    font-size: 28px;
+}
+
+#projectList {
+    margin-top: 20px
+}
+
+#projects a {
+    text-decoration: none;
+    text-align: center;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-weight: bold;
+    font-size: 30px;
+    border-bottom: 2px solid black;
+}
+/* I'll have to refine how I placed the project screenshots on this page. */
```

Comment:
This is a good example of a comment/note that you should use throughout your code. Its little things like this or a note about how your page is meant to display(header section, sidebar content, navbar, etc). Just more of a reference for you to quickly know where something is or something you'd like to revisit